ui/pages/confirmations/utils/confirm.ts

```diff
@@ -119,3 +122,39 @@ export const toPunycodeURL = (urlString: string): string | undefined => {
 export const stripProtocol = (urlString: string): string => {
   return urlString.replace(/^\w+:\/\//u, '');
 };
+
+const validSwapBatchTransactionCommands = [
+  '0x3593564c',
+  '0x87517c45',
+  '0x095ea7b3',
+];
```

Comment:
These are magic hex... we should give them meaning 😅 (either through a code comment or independent variables naming)

Reply:
Let me update the PR.

Reply:
PR is updated